python/chronos/src/bigdl/chronos/forecaster/nbeats_forecaster.py

```diff
@@ -0,0 +1,141 @@
+                 seed=None,
+                 distributed=False,
+                 workers_per_node=1,
+                 distributed_backend="torch_distributed"):
+        """
+        Build a NBEATS Forecaster Model.
+
+        :param past_seq_len: Specify the history time steps (i.e. lookback).
+        :param future_seq_len: Specify the output time steps (i.e. horizon).
+        :param stack_types: Specifies the type of stack,
+               including "generic", "trend", "seasnoality".
+               This value defaults to ("generic", "generic").
+        :param nb_blocks_per_stack: Specify the number of blocks
+               contained in each stack, This value defaults to 3.
+        :param thetas_dim: Expansion Coefficients of Multilayer FC Networks.
+               if type is "generic", Extended length factor, if type is "trend"
+               then polynomial coefficients, if type is "seasonality"
+               expressed as a change within each step.
+        :param share_weights_in_stack: Share block weights for each stack.,
+               This value defaults to False.
+        :param hidden_layer_units: Number of fully connected layers with per block.
+               This values defaults to 256.
+        :param nb_harmonics: Only available in "seasonality" type,
+               specifies the time step of backward, This value defaults is None.
+        :param dropout: Specify the dropout close possibility
+               (i.e. the close possibility to a neuron). This value defaults to 0.1.
+        :param optimizer: Specify the optimizer used for training. This value
+               defaults to "Adam".
+        :param loss: Specify the loss function used for training. This value
+               defaults to "mse". You can choose from "mse", "mae" and
+               "huber_loss".
+        :param lr: Specify the learning rate. This value defaults to 0.001.
+        :param metrics: A list contains metrics for evaluating the quality of
+               forecasting. You may only choose from "mse" and "mae" for a
+               distributed forecaster. You may choose from "mse", "mae",
+               "rmse", "r2", "mape", "smape", for a non-distributed forecaster.
+        :param seed: int, random seed for training. This value defaults to None.
+        :param distributed: bool, if init the forecaster in a distributed
+               fashion. If True, the internal model will use an Orca Estimator.
+               If False, the internal model will use a pytorch model. The value
+               defaults to False.
+        :param workers_per_node: int, the number of worker you want to use.
+               The value defaults to 1. The param is only effective when
+               distributed is set to True.
+        :param distributed_backend: str, select from "torch_distributed" or
+               "horovod". The value defaults to "torch_distributed".
+        """
+        # TODO Currently generic mode not support distributed.
```

Comment:
raise a issue once this PR is merged.

Reply:
For "not support distributed." do you mean it does not support `distributed=True`(real distributed mode) or the multi-process training we automatically triggered when user has more than 8 cores?

Reply:
`distributed`=True is not supported, only happens when setting `stack_types`=("generic", "generic").
If `stack_types`=("generic", "trend" or "seasonality") the above problem will not occur.
I will modify the description to "not support distributed=True" in the next commit.

Reply:
It seems to be an orca issue (related to `find_unused_parameter` or something like it), we can raise an orca issue after this PR.

Reply:
ok, l will describe the problem in detail.